app/uk/gov/hmrc/cdsreimbursementclaimfrontend/connectors/RejectedGoodsSingleClaimConnector.scala

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.cdsreimbursementclaimfrontend.connectors
+
+import akka.actor.ActorSystem
+import cats.syntax.eq._
+import com.google.inject.ImplementedBy
+import com.google.inject.Inject
+import play.api.libs.json.Format
+import play.api.libs.json.Json
+import uk.gov.hmrc.cdsreimbursementclaimfrontend.journeys.RejectedGoodsSingleJourney
+import uk.gov.hmrc.cdsreimbursementclaimfrontend.utils.HttpResponseOps._
+import uk.gov.hmrc.http.HeaderCarrier
+import uk.gov.hmrc.http.HttpClient
+import uk.gov.hmrc.http.HttpReads.Implicits._
+import uk.gov.hmrc.http.HttpResponse
+import uk.gov.hmrc.play.bootstrap.config.ServicesConfig
+
+import javax.inject.Singleton
+import scala.concurrent.ExecutionContext
+import scala.concurrent.Future
+import scala.concurrent.duration._
+import play.api.Configuration
+
+@ImplementedBy(classOf[RejectedGoodsSingleClaimConnectorImpl])
+trait RejectedGoodsSingleClaimConnector {
+
+  def submitClaim(claimRequest: RejectedGoodsSingleClaimConnector.Request)(implicit
+    hc: HeaderCarrier
+  ): Future[RejectedGoodsSingleClaimConnector.Response]
+}
+
+object RejectedGoodsSingleClaimConnector {
+
+  final case class Request(claim: RejectedGoodsSingleJourney.Output)
+  final case class Response(caseNumber: String)
+  final case class Exception(msg: String) extends scala.RuntimeException(msg)
+
+  implicit val requestFormat: Format[Request]   = Json.format[Request]
+  implicit val responseFormat: Format[Response] = Json.format[Response]
+}
+
+@Singleton
+class RejectedGoodsSingleClaimConnectorImpl @Inject() (
+  http: HttpClient,
+  servicesConfig: ServicesConfig,
+  configuration: Configuration,
+  val actorSystem: ActorSystem
+)(implicit
+  ec: ExecutionContext
+) extends RejectedGoodsSingleClaimConnector
+    with Retries {
+
+  import RejectedGoodsSingleClaimConnector._
+
+  val baseUrl: String                     = servicesConfig.baseUrl("cds-reimbursement-claim")
+  val claimUrl: String                    = s"$baseUrl/claims/rejected-goods-single"
```

Comment:
Hi, I think the way it was done it missing `/cds-reimbursement-claim`, so the final URl should be
`$baseUrl/cds-reimbursement-claim/claims/rejected-goods-single`

Because of routes file
`->          /cds-reimbursement-claim                app.Routes`

Reply:
BTW I would move
`$baseUrl/cds-reimbursement-claim` into config servicesConfig.baseUrl("cds-reimbursement-claim")

Reply:
good spot